iphone/Classes/TiUITabGroup.m

```diff
@@ -42,6 +42,12 @@ - (UITabBarController *)tabController
     controller.delegate = self;
     controller.moreNavigationController.delegate = self;
     [TiUtils configureController:controller withObject:self.proxy];
+    if ([TiUtils isIOSVersionOrGreater:@"15.0"]) {
```

Comment:
I _think_ this section will need wrapping in a `#if IS_SDK_IOS_15` block. I believe that's the cause of the build failure in the tests on Jenkins